Tests/GitHubAPI/API.swift

```diff
@@ -16,23 +18,23 @@ public final class RepositoryQuery: GraphQLQuery {
       GraphQLField("repository", arguments: ["owner": "apollographql", "name": "apollo-ios"], type: .object(Repository.selections)),
     ]
 
-    public var snapshot: Snapshot
+    public private(set) var resultMap: ResultMap
```

Comment:
This change doesn't seem to be related to the operation identifiers - do you know why this is coming through here? It does seem like it was already changed in the star wars example

Reply:
Yes, you are right. It's not related to operation identifiers.
The changes is related to the fix of CodeGen. Once CodeGen is running successfully, this files updated. I guess GitHubAPI have been failure for awhile which made this file outdated.

Reply:
OK thanks!